src/faldbt/project.py

```diff
@@ -365,6 +365,7 @@ def _find_features(self) -> List[Feature]:
             filter(
                 # Find models that have both feature store and column defs
                 lambda model: keyword in model.meta
+                and isinstance(model.meta[keyword], dict)
```

Comment:
This is a good one. But what if the user passes a string by mistake? Shouldn't we let the user know?

Reply:
Would you just raise an exception?

Reply:
Yes, I think so.